specification/network/resource-manager/Microsoft.Network/stable/2020-05-01/firewallPolicy.json

```diff
@@ -623,9 +623,9 @@
           "description": "ThreatIntel Whitelist for Firewall Policy.",
           "$ref": "#/definitions/FirewallPolicyThreatIntelWhitelist"
         },
-        "intrusionSystemMode": {
-          "description": "The operation mode for Intrusion system.",
-          "$ref": "#/definitions/FirewallPolicyIntrusionSystemMode"
+        "intrusionSystem": {
```

Comment:
Any example change needed?

Reply:
we still didn't add examples - this feature is currently available only by feature flag